src/ansys/geometry/core/math/vector.py

```diff
@@ -415,7 +425,7 @@ def __sub__(self, other: "Vector2D") -> "Vector2D":
         """Subtraction operation overload for 2D vectors."""
         return np.subtract(self, other).view(Vector2D)
 
-    def __mod__(self, other: "Vector2D") -> "Vector2D":
+    def __mod__(self, other: "Vector2D"):
```

Comment:
For my own knowledge, why not mentioning the returned type?

Reply:
It's a built-in Numpy type (np.ndarray).  IDEs detect it by default. Also, it will no longer be a 2D vector. The cross product of 2D vectors is a number. However, the default behavior of Numpy is not to return a single number but a 1D Numpy array. See the following example:

```python
x = [1,2]
y = [4,5]
np.cross(x, y)
```

Reply:
I decided to add the typehints in the end - even if they are not necessary, that way our library is more explicit. Thanks again @MaxJPRey for the suggestion!

Reply:
Got it.